x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/GetDataFrameTransformsStatsAction.java

```diff
@@ -43,6 +47,11 @@ public Response newResponse() {
 
     public static class Request extends BaseTasksRequest<Request> {
         private String id;
+        private PageParams pageParams = PageParams.defaultParams();
+
+        public static final int MAX_SIZE_RETURN = 1000;
```

Comment:
Using 1k here to keep in line with the config setting. Unsure if we want to make that the standard for dataframes as a whole. If so, I can move it out to be a globally shared constant.